### DISCUSSION TITLE
How to skip a bad library version

### DISCUSSION THREAD

**Discussion by alexklibisz** [scala-steward-org/scala-steward#3030] [Q&A]
Say I know that `"com.foo" %% "library" % "1.2.3"` is buggy.

Is there any way for me to tell scala-steward to not upgrade to this specific version?

At first glance, it seems like updating `updates.ignore` to include this (org, artifact, version) might solve this problem. However the actual behavior seems to be that scala-steward will ignore updates for any project that is already on this (org, artifact, version).

**Comment by mzuehlke** [comment]
Is this the behaviour your experienced, or what you expect from the documentation ?
The documentation can be interpreted differently:

>  The dependencies which match the given pattern are NOT updated.

But the configuration (and the documentation) is about candidates for a potential update

**Reply by alexklibisz** [reply]
The behavior I experience is: an _existing_ dependency which matches the given pattern is not documented. 

The first time I read the documentation, I came away with the other understanding: a _potential_ dependency which matches the given pattern would be ignored. That also happened to be the behavior I was looking for, which probably biased my interpretation of the docs.

Is there any way to get this behavior? Or maybe I can make a PR to get this behavior?
